wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialise in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
+
+## osu! Development server
+
+The **osu! Development** server — also known as **osu!dev** or just **osu!** — is the game's official Discord server. It is meant to be the main place to view and discuss osu!'s ongoing developments, a place to contribute to the codebase, and a place where a plethora of community projects are organised in!
```

Comment:
why is there a description both here and in the table

the one in the table is written better imo. but it should be moved up here probably

Reply:
As a solo-element on a table without description it looks kind of weird. See the latest commit once that goes up.

Resolving this, but a solution on how could this be formatted would be appreciated. Maybe not having the description as a header and having it on the table itself would help?